test_views.py

```diff
@@ -65,16 +65,17 @@ def test_invalid_weather_url_empty(self):
         result = self.client.get('/weather_forecast?address=')
         self.assertEqual(result.status_code, 400)
 
-    @patch('app.geocode_api_parser.Geocode.is_place_query_valid')
-    def test_invalid_weather_url_invalid_address(self, is_place_query_valid,):
+    @patch('app.geolocation_api_parser.Geolocation.is_place_query_valid')
+    def test_invalid_weather_url_invalid_address(self, is_place_query_valid):
         is_place_query_valid.return_value = False
         result = self.client.get('/weather_forecast?address=pppppppppp')
         self.assertTrue(is_place_query_valid.called)
         self.assertEqual(result.status_code, 400)
 
     @patch('app.weather_parser.WeatherForecast.generate_forecast')
-    @patch('app.geocode_api_parser.Geocode.get_address_query')
-    def test_valid_weather_url_valid_address(self, get_address_query, generate_forecast):
+    @patch('app.geolocation_api_parser.Geolocation.get_local_time')
```

Comment:
nope, get_local_time is doing too much than to be patched. patch surgically, only the minimum.